fil_fungible_token/src/token/mod.rs

```diff
@@ -376,15 +443,32 @@ mod test {
         assert_eq!(total_supply, TokenAmount::from(1_000_000));
     }
 
+    #[test]
+    fn it_fails_to_mint_if_receiver_hook_aborts() {
+        let mut token = new_token();
+
+        let treasury = 1;
+        // force hook to abort
+        token
+            .mint(TOKEN_ACTOR_ADDRESS, treasury, &TokenAmount::from(1_000_000), "abort".as_bytes())
+            .unwrap_err();
```

Comment:
As noted above, I'd like to see tests that the hook was invoked with expected parameters, i.e. with a mock.